src/amitaRJun20/PalindromeNumber.java

```diff
@@ -0,0 +1,25 @@
+package amitaRJun20;
+
+public class PalindromeNumber {
+
+	void isPalindromeNumber(int num) {
```

Comment:
Suggestion: Can add a comment to describe the method. It will be helpful when we are trying to find code or some else is trying to understand your code base which is huge.
Rest all is perfect.

Reply:
Sure Mahesh...Thanks for your feedback...i will keep in mind from next time onwards...